api/transactions/withdraw.js

```diff
@@ -1,9 +1,23 @@
 const { connectToDatabase } = require("../../lib/mongoose");
 const Bet = require("../../models/bet.model");
-const Prediction = require("../../models/prediction.model");
 const solanaWeb3 = require("@solana/web3.js");
 const { clusterApiUrl, Connection, Keypair, PublicKey, sendAndConfirmTransaction, SystemProgram, Transaction, LAMPORTS_PER_SOL } = solanaWeb3;
 
+const checkBalanceAndFund = async (connection, publicKey) => {
+    try {
+        const rawBalance = await connection.getBalance(publicKey);
+        const formattedBalance = rawBalance/LAMPORTS_PER_SOL;
+        console.log('current balance is:', formattedBalance);
+    
+        if(formattedBalance < 10) {
```

Comment:
`formattedBalance && formattedBalance < 10` in case formattedBalance is undefined or something funny.
What do you think?

Reply:
no need. If something goes wrong (even if it is undefined) it will be handled by the catch handle